cpu/msp430_common/cpu.c

```diff
@@ -55,7 +55,8 @@ void *thread_isr_stack_start(void)
 
 NORETURN void cpu_switch_context_exit(void)
 {
-    sched_active_thread = sched_threads[0];
+    /* FIXME: This line seems to be unneeded. Is this true? */
+    /* sched_active_thread = sched_threads[0]; */
```

Comment:
TODO: Does this line of code have a purpose? `sched_run()` will set `sched_active_thread` anyway, right?